examples/flutter-drm-eglstream-backend/command_options.h

```diff
@@ -0,0 +1,362 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
```

Comment:
Can you share this header source in each programs?

Reply:
What does it mean?

Reply:
I found same/duplicated header code in each programs. If each programs include one header, it will be easy to maintain the code.

Reply:
The master files including main.cc are located in the under src/templates/app directory. On the other hand, the examples directory is just for Flutter example projects. The files which are in the under examples directory can be modified freely by users. I think it's a waste, but it's good for users to be separated individually for stand-alone projects and user customization. How do you think?

Reply:
Your modification is ok for me. I can understand the benefits to include the same file in the sample code.